Plugins/GitUIPluginInterfaces/GitRefsFlags.cs

```diff
@@ -1,18 +1,21 @@
-﻿namespace GitCommands.Git.Commands
+﻿namespace GitUIPluginInterfaces
 {
     /// <summary>
     /// Enums requestable in GitRefs() (multiple names can be appended)
     /// Compare to <see ref="GitRefType"/> for actual values of parsed GitRefs
     /// </summary>
-    public enum GetRefsEnum
+    public enum GitRefsFlags
```

Comment:
https://docs.microsoft.com/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations
```suggestion
    [Flags]
    public enum GitRefs
```

Reply:
Can the guidelines be summarized somewhere of the current codestyle?
Even more important for new contributors.

Changing this, but the name is very close to the existing class GitRef, this is confusing.

Reply:
> Can the guidelines be summarized somewhere of the current codestyle?

For many years in some shape or form I've been using Microsoft .NET design guidelines - originally [posted by Brad Abrams](https://docs.microsoft.com/en-us/archive/blogs/brada/internal-coding-guidelines) and the [current official](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/). 

A lot of styling guidelines can now be enforced with the help of StyleCop and Roslyn analyzers (and that what we've been doing for some time), though some things are hard to enforce automatically (like naming).

Reply:
> Changing this, but the name is very close to the existing class GitRef, this is confusing.

Perhaps we need to revisit the naming of the class or both. E.g. the enum can also be named `GitRefTypes`...